tests/integ/modin/test_apply_snowpark_python_functions.py

```diff
@@ -89,3 +89,14 @@ def test_apply_snowflake_cortex_summarize():
     summary = s.apply(snowflake_cortex_summarize).iloc[0]
     # this length check is to get around the fact that this function may not be deterministic
     assert 0 < len(summary) < len(content)
+
+
+@sql_count_checker(query_count=1)
+@pytest.mark.skip("SNOW-1758914 snowflake.cortex.sentiment error on GCP")
```

Comment:
Have you checked this?

Reply:
Yes, this test passes locally but fails the gcp precommit test here previously when the test was not skipped